Creiwry/irl-meeting-app
# UI: add controller pages_controller and related view welcome

### Ticket Link
Add a button from landing to registrations#new

### User Story
(https://templatethpprojetfinal.atlassian.net/browse/THPPROJ-37)

### What does this PR do?
Create a controller called 'pages_controller'
Create the view 'welcome' in 'views/pages/ with a button displayed in the middle of the screen, and a background image.
No link with back-end for the time being, just display
The page is responsive to any size of screens.

### Where should the reviewer start?
(Your answer here.)

### How should this be manually tested?
Load the page welcome and resize your screen, image should always cover the entire screen, and button should remain centered vertically & horizontally.

### Screenshots (if appropriate)
![image](https://github.com/Creiwry/irl-meeting-app/assets/137895449/7606f93b-fbef-4112-ac3e-dd196ff1e04d)


### Are there any breaking changes?
No
